frontend/apps/reader/modules/readertypography.lua

```diff
@@ -701,7 +701,7 @@ end
 -- in book settings, no default lang, and book has some language defined.
 function ReaderTypography:onReadSettings(config)
     -- Migrate old readerhyphenation setting, if one was set
-    if not config:readSetting("text_lang") and config:readSetting("hyph_alg") then
+    if config:hasNot("text_lang") and config:hasNot("hyph_alg") then
```

Comment:
**Error here**: `hasnot and has` translated to `hasnot and hasnot`

Reply:
Oh, should have been obvious given the following line, thanks ;).